packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -15,8 +15,8 @@ import { Changeset, Transform } from './dataflow';
 import { Scene } from './scene';
 import { LoggerInterface } from 'vega-util';
 
-// TODO
-export type Runtime = any;
+export { Runtime } from './runtime';
+import { Runtime } from './runtime';
```

Comment:
```suggestion
```

Reply:
The `Runtime` type is used in the `parse` function in this file, so I found I had to import it as well as export.

Reply:
I see. What I usually do is import and then do something like `export cost foo`. But I see that's not our style here so ignore my comment.